eng/targets/TargetFrameworks.props

```diff
@@ -15,7 +15,7 @@
     <NetRoslyn>net8.0</NetRoslyn>
     <NetRoslynAll>net7.0;net8.0</NetRoslynAll>
     <NetVS>net8.0</NetVS>
-    <NetVSCode>net7.0</NetVSCode>
+    <NetVSCode>net8.0</NetVSCode>
```

Comment:
Not saying it has to be part of this PR, but this would mean you could delete that load-all-assemblies-upfront hack we have to work around the 7.0.4 runtime bug.

Reply:
I'll do that in a followup to avoid conflicts with my other pending PR there.